src/statsig/index.ts

```diff
@@ -84,7 +84,7 @@ export function getDynamicConfigParams<T extends Record<string, StatsigParameter
   }
 }
 
-export function getFeatureGate({ featureGateName }: { featureGateName: StatsigFeatureGates }) {
+export function getFeatureGate(featureGateName: StatsigFeatureGates) {
```

Comment:
realized that since there was only one argument the object spread param was a bit redundant